dask/optimize.py

```diff
@@ -427,6 +428,179 @@ def fuse_getitem(dsk, func, place):
                            lambda a, b: tuple(b[:place]) + (a[2], ) + tuple(b[place + 1:]))
 
 
+def fuse_reductions(dsk, keys=None, ave_width=2, max_depth_new_edges=None,
+                    max_height=None, max_width=None):
```

Comment:
What's the order of precedence?  What if the parameter exists in `_globals` but is `None`?  Is there common practice here yet?